### PR TITLE
Added safty checks and tooltips

### PR DESCRIPTION
**Pull request by SidaDan**
Hi :)  I hope you find my changes usefull.

Changes:
-Added Tooltip Combobox Font type and Font size
-Overwrite method processWindowEvent to detect window closing event
-Added safty query to save the file before exit
or the user selects "newfile" (only if the user has written something).
-Added safty query if the user presses the clearButton

### CODE REVIEW

File: src/simplejavatexteditor/UI.java [hunk 1]
```diff
@@ -300,76 +307,86 @@ public void actionPerformed(ActionEvent ev) {
...
         });
 
         //FONT FAMILY SETTINGS SECTION END
-
-
         //FONT SIZE SETTINGS START
-
         fontSize = new JComboBox();
 
-            for( int i = 5 ; i <= 100 ; i++)
```
Comment (pH-7) [inline src/simplejavatexteditor/UI.java:350]: Thanks for the indentation fixes 👍

**Comment by pH-7** [comment]
Hi @SidaDan!
Thanks a lot for those improvements! Highly appreciate!
Did you test your changes with Java 7/8 (especially for the tooltip feature). Is everything OK..?

Thanks!

**Comment by SidaDan** [comment]
Hello pH-7,
I tested it with java 8. All changes works fine. 
Regards

**Comment by pH-7** [comment]
Brillant @SidaDan

**Pull request merged**